orchestrator/gravity_utils/src/types/ethereum_events.rs

```diff
@@ -841,7 +841,7 @@ mod tests {
 
     use super::*;
 
-    const FUZZ_TIMES: u64 = 100_000;
```

Comment:
why was this reduced? `SendToCosmosEvent` is fairly complex and indirectly tests alot of things, `FUZZ_TIMES` should be increased to whatever is bearable long term, e.g. what would take about a second or two.

Reply:
It was reduced because of the test execution time, it took 1h. I changed it to 10k now. It should be enough now.